src/app/components/ElectronicDelivery/ElectronicDeliveryForm.jsx

```diff
@@ -150,11 +197,19 @@ class ElectronicDeliveryForm extends React.Component {
               </div>
             </div>
           </div>
-
+        </fieldset>
+        <fieldset>
+          <legend>
+            <h4>Additional Details</h4>
```

Comment:
@courtneymcgee Is Additional Details supposed to be an `h3` or an `h4`? It's currently an `h4`, but "Required Fields" is an `h3`.

Reply:
This is also the way it is in the Design Toolkit.

Reply:
Additional Details should be h3